config/environments/development.rb

```diff
@@ -39,6 +43,9 @@
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
+  # Highlight code that triggered database queries in logs.
+  config.active_record.verbose_query_logs = true
+
```

Comment:
Background jobs are not running at all.  Please add `config.active_job.queue_adapter = :inline`

Reply:
Got em, thanks.